.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: fhir2dataset
+
+on: [push]
+
+jobs:
+  build:
+    name: Lint and test
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Lint with flake8
+        run: |
+          python -m pip install flake8
+          # stop the build if there are Python syntax errors or undefined names
+          flake8 fhir2dataset --count --select=E9,F63,F7,F82 --show-source --statistics
+          flake8 fhir2dataset --count --max-complexity=10 --max-line-length=100 --statistics
+
+  publish:
+    name: Build and publish
+    runs-on: ubuntu-18.04
+    needs: build
+    if: github.ref == 'refs/heads/master'
+    strategy:
+      matrix:
+        python-version: ["3.7", "3.8"]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
```

Comment:
have you registered the secrets in github ?

Reply:
Yes! It's done